### PR TITLE
Support database API

### PR DESCRIPTION
**Pull request by jaime0815**
/kind improvement

### CODE REVIEW

**Review by XuanYang-cn — CHANGES_REQUESTED** [review]

File: pymilvus/grpc_gen/milvus_pb2.py [hunk 1]
```diff
@@ -11,8 +11,8 @@
 _sym_db = _symbol_database.Default()
 
 
-from . import common_pb2 as common__pb2
-from . import schema_pb2 as schema__pb2
+from . from . from . from . import common_pb2 as common__pb2
```
Comment (longjiquan) [inline pymilvus/grpc_gen/milvus_pb2.py:14]: Use `make gen_proto` to update the generated py files.

File: pymilvus/grpc_gen/schema_pb2.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 _sym_db = _symbol_database.Default()
 
 
-from . import common_pb2 as common__pb2
+from . from . from . from . import common_pb2 as common__pb2
```
Comment (longjiquan) [inline pymilvus/grpc_gen/schema_pb2.py:14]: Use `make gen_proto` to update the generated py files.

File: pymilvus/grpc_gen/schema_pb2.pyi [hunk 3]
```diff
@@ -1,4 +1,4 @@
-from . import common_pb2 as _common_pb2
+from . from . from . from . import common_pb2 as _common_pb2
```
Comment (longjiquan) [inline pymilvus/grpc_gen/schema_pb2.pyi:1]: Use `make gen_proto` to update the generated py files.

File: pymilvus/__init__.py [hunk 4]
```diff
@@ -84,7 +84,7 @@
     'SearchResult', 'Hits', 'Hit', 'Replica', 'Group', 'Shard',
     'FieldSchema', 'CollectionSchema',
     'SearchFuture', 'MutationFuture',
-    'utility', 'DefaultConfig', 'ExceptionsMessage', 'MilvusUnavailableException', 'BulkInsertState',
+    'utility', 'db', 'DefaultConfig', 'ExceptionsMessage', 'MilvusUnavailableException', 'BulkInsertState',
```
Comment (longjiquan) [inline pymilvus/__init__.py:87]: why not export these three apis? `db` here seems strange.

**Review by longjiquan — CHANGES_REQUESTED** [review]

**Comment by longjiquan** [comment]
Please also add this feature to master branch.

File: pymilvus/orm/connections.py [hunk 5]
```diff
@@ -214,7 +216,7 @@ def remove_connection(self, alias: str):
         self.disconnect(alias)
         self._alias.pop(alias, None)
 
-    def connect(self, alias=Config.MILVUS_CONN_ALIAS, user="", password="", **kwargs):
+    def connect(self, alias=Config.MILVUS_CONN_ALIAS, user="", password="", db_name="default", **kwargs):
```
Comment (czs007) [inline pymilvus/orm/connections.py:219]: It is recommended to fill in empty "or None here, and convert it to default at the server.

File: pymilvus/client/grpc_handler.py [hunk 6]
```diff
@@ -771,6 +775,28 @@ def get_loading_progress(self, collection_name, partition_names=None, timeout=No
             raise MilvusException(response.status.error_code, response.status.reason)
         return response.progress
 
+    @retry_on_rpc_failure()
+    def create_database(self, db_name, timeout=None):
+        request = Prepare.create_database_req(db_name)
+        status = self._stub.CreateDatabase.future(request, timeout=timeout).result()
```
Comment (longjiquan) [inline pymilvus/client/grpc_handler.py:781]: Do not use `future`.

File: pymilvus/client/grpc_handler.py [hunk 7]
```diff
@@ -771,6 +775,28 @@ def get_loading_progress(self, collection_name, partition_names=None, timeout=No
...
+        if status.error_code != 0:
+            raise MilvusException(status.error_code, status.reason)
+
+    @retry_on_rpc_failure()
+    def drop_database(self, db_name, timeout=None):
+        request = Prepare.drop_database_req(db_name)
+        status = self._stub.DropDatabase.future(request, timeout=timeout).result()
```
Comment (longjiquan) [inline pymilvus/client/grpc_handler.py:788]: Do not use `future`.

File: pymilvus/client/grpc_handler.py [hunk 8]
```diff
@@ -771,6 +775,28 @@ def get_loading_progress(self, collection_name, partition_names=None, timeout=No
...
+        if status.error_code != 0:
+            raise MilvusException(status.error_code, status.reason)
+
+    @retry_on_rpc_failure()
+    def list_database(self, timeout=None):
+        request = Prepare.list_database_req()
+        response = self._stub.ListDatabases.future(request, timeout=timeout).result()
```
Comment (longjiquan) [inline pymilvus/client/grpc_handler.py:795]: Do not use `future`.

**Review by longjiquan — CHANGES_REQUESTED** [review]

**Review by longjiquan — APPROVED** [review]
/lgtm

**Comment by longjiquan** [comment]
Please also add this feature into master branch.

**Pull request merged**